inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -506,10 +506,20 @@ bool is_exec_graph(const ngraph::Function& f) {
     return false;
 }
 
+bool has_dynamic_output(std::shared_ptr<Node> n) {
+  for (size_t i = 0; i < n->get_output_size(); i++) {
+    if (n->get_output_partial_shape(i).is_dynamic()) {
```

Comment:
nit: incorrect indent

Reply:
Can we expect .clang-format for IE anytime soon ? If you create such ticket I can happily take it on the next release :)

Reply:
@jdanoeck It is a big story. Let's discuss it offline.